### PR TITLE
Modelo do milho

### LINKED ISSUE #5: Modelo do milho
# Descrição
Em #6, encontramos alguns datasets bem interessantes. Dentre eles, há o milho,  já é possivel criar uma rede convolucional simples

# Tarefas
- [x] criar uma rede neural convolucional

# Critério de aceitação
- [x] Acurácia do treinamento em média de 90%

### PR DESCRIPTION
**Pull request by thiagohdaqw**
# Descrição
Foi adicionado o novo modelo do milho, isso foi necessário porque, mesmo com uma suposta acurácia de 99%, não estava acertando as doenças. Acredito que esse overfitting estava ocorrendo por causa da não diversidade do dataset, aparentemente as imagens dos testes também estavam no de treino, mesmo pegando de dois banco de dados supostamente diferentes. Agora consegui diversicar esse dataset e consegui uma acurácia de ~90%.

# Resolve
#5 

# Capturas de Tela
image
image

# Tarefas realizadas
* Diversificação do dataset
* Adição de camada de Dropout e Batchnormalization à CNN
* Teste simples do modelo

### CODE REVIEW

**Comment by thiagohdaqw** [comment]
Dataset retirado de https://github.com/pratikkayal/PlantDoc-Dataset/tree/master

**Review by RcleydsonR — APPROVED** [review]
Perfeito Thiago, a Acurácia até que está okay, deve cumprir o propósito do nosso projeto. Bom trabalho !
